### ISSUE TITLE
100.64.0.0/10 is not interpreted as a private IP range

### ISSUE THREAD

**Issue by jqueuniet** [pydio/cells#430]
Hello,

I'm trying to run pydio cells in a Kubernetes cluster using 100.64.0.0/16 for pods and 100.65.0.0/16 for services. Those ranges are part of the 100.64.0.0/10 address block, which was originally allocated for carrier-grade NAT, but is also usable as general-purpose private space as it shares the same properties as the traditional RFC1918 IPv4 address blocks.

Would it be possible to avoid flagging those IPv4 as public addresses, as pydio cells currently does?

```
2022-09-17T09:14:58.840Z	INFO	pydio.grpc.data.sync.personal	Warning: no private IP detected for binding broker. Will bind to 100.64.166.148, which may give public access to the broker.
```

**Comment by bsinou** [comment]
Hello,

We have improved this part in the server and as from v4, we do not need an additional private IP address .
Could you please have a look at the latest v4.0.x version and let us know if you issue is gone ? 

Many thanks

**Comment by jqueuniet** [comment]
Indeed, I don't seem to get this error anymore using PyDio Cells v4
